### PR TITLE
Update create-retention-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line to reduce ambiguity on which recipient type  the retention policy can be set

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5132

**Comment by msbemba** [comment]
@cabailey I have made the changes on this document as it is more appropriate to add the information here. If you think I need to add it only here -https://docs.microsoft.com/en-us/microsoft-365/compliance/retention-policies-exchange?view=o365-worldwide or on both the documents, let me know.

Thanks
Sri

File: microsoft-365/compliance/create-retention-policies.md [hunk 1]
```diff
@@ -179,14 +179,16 @@ Use the following instructions for retention policies that apply to any of these
 
 #### Configuration information for Exchange email and Exchange public folders
 
-The **Exchange email** location supports retention for users' email, calendar, and other mailbox items, by applying retention settings at the level of a mailbox.
+The **Exchange email** location supports retention for users' email, calendar, and other mailbox items, by applying retention settings at the level of a mailbox. Retention Policy can only be applied to a Licensed Mailbox, distribution group or mail-enabled security group (all mailboxes that are currently members of the group).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-retention-policies.md]:
```suggestion
The **Exchange email** location supports retention for users' email, calendar, and other mailbox items, by applying retention settings at the level of a mailbox. Retention policies can only be applied to licensed mailboxes, distribution groups, or mail-enabled security groups (to all mailboxes that are currently members of the group).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cabailey Please merge this PR. Thanks

**Comment by cabailey** [comment]
#hold-off  This statement introduces ambiguity and needs to be verified by engineering.  Please do not merge while I investigate.

**Comment by cabailey** [comment]
Closing as discussed in the linked GitHub Issue.  Contact me internally if questions.

**Pull request closed without merging**